docs/Antora/modules/howto_guides/pages/howto_configure_instance.adoc

```diff
@@ -0,0 +1,291 @@
+[tabs]
+====
+Cpp::
++
+[source,cpp]
+----
+#include <opendaq/opendaq.h>
+#include <iostream>
+
+int main(int argc, const char* argv[])
+{
+    daq::InstanceBuilderPtr instanceBuilder = daq::InstanceBuilder();
+                        .addLoggerSink(StdOutLoggerSink())
+                        .setSinkLogLevel(BasicFileLoggerSink("logfile.log"),  daq::LogLevel::Error);
+                        .setComponentLogLevel("Instance", daq::LogLevel::Warn);
+                        .setComponentLogLevel("Some Module", daq::LogLevel::Off);
+
+    daq::InstancePtr instance = instanceBuilder.build();
+
+    return 0;
+}
+----
+====
+
+An advanced way of configuring the Instance logger is to configure the logger object and set it with the `setLogger` method.
+[tabs]
+====
+Cpp::
++
+[source,cpp]
+----
+#include <opendaq/opendaq.h>
+#include <iostream>
+
+
+int main(int argc, const char* argv[])
+{
+    daq::ListPtr<ILoggerSink> sinks = daq::List<ILoggerSink>(StdOutLoggerSink());
+    daq::LoggerPtr logger = daq::LoggerWithSinks(sinks, daq::LogLevel::Warn);
+    daq::InstanceBuilderPtr instanceBuilder = daq::InstanceBuilder().setLogger(logger);
+
+    daq::InstancePtr instance = instanceBuilder.build();
+
+    return 0;
+}
+----
+====
+
+[NOTE]
+====
+Alternatively, a developer can create their own implementation of the logger, which has to inherit the ILogger interface.
+====
+
+== Configure openDAQ(TM) Instance Module Manager
+By default, the Instance Module Manager uses the current path for loading modules. In the Instance Builder, developers can set a custom module path or replace the default Module Manager with a custom implementation inherited from IModuleManager.
+[tabs]
+====
+Cpp::
++
+[source,cpp]
+----
+#include <opendaq/opendaq.h>
+#include <iostream>
+
+
+int main(int argc, const char* argv[])
+{
+    daq::InstanceBuilderPtr instanceBuilder = daq::InstanceBuilder().setModulePath("/path/to/modules");
+    daq::InstancePtr instance = instanceBuilder.build();
+
+    return 0;
+}
+----
+Overriding module path
+====
+
+[tabs]
+====
+Cpp::
++
+[source,cpp]
+----
+#include <opendaq/opendaq.h>
+#include <iostream>
+
+class CustomMuduleManagerImpl : public ImplementationOfWeak<IModuleManager>
+{
+    CustomMuduleManagerImpl() = default;
+    // implementation
+} 
+
+int main(int argc, const char* argv[])
+{
+    daq::ModuleManagerPtr moduleManager = daq::ModuleManagerPtr(CustomMuduleManagerImpl());
+    daq::InstanceBuilderPtr instanceBuilder = daq::InstanceBuilder().setModuleManager(moduleManager);
+    daq::InstancePtr instance = instanceBuilder.build();
+
+    return 0;
+}
+----
+Setting custom module manager
+====
```

Comment:
This code is out-of-scope imho. I'd just leave it out, as custom implementations of interfaces are not yet documented at all and are not the target of this how-to.